promptops/promptops/skill_export.py

```
@@ -92,16 +92,19 @@ def parse_prompt_yaml(raw_content: str) -> tuple:
-            is_skill = True
-    return is_skill
+        tags = extract_tags(raw_data)
+        if "skill" in tags:
+            return True
+            
+    # Fallback to raw text extraction
+    text_tags = extract_tags_from_text(raw_content)
+    if "skill" in text_tags:
+        return True
+        
+    return False
```

Ruff (0.15.15)

[warning] Return the condition `"skill" in text_tags` directly

Replace with `return "skill" in text_tags`

(SIM103)